### ISSUE TITLE
Midterm Report Peer Review

### ISSUE THREAD

**Issue by vladiatrinh** [talvarez0507/orie4741-fp#5]
# Summary
The dataset used in this project is the Football Dataset. This team is aiming to predict soccer game results using match details, full-time and half-time details, and standard game odds. To fit a baseline linear model, they did some feature engineering such as one-hot encoding and removing features will large amounts of missing values. Using a multinomial logistic regression model, they achieved a baseline prediction accuracy of 60.99% over the train data and 57.43% over the test data.

# Three Things I Like
- I like that you are using several datasets to train your model. It is particularly challenging to scrape the web for clean data and combine multiple sources to create one cohesive dataset. I'm sure it will improve the predictive power of your model.
- I like that you already used regularization in your logistic regression model. This helps prevent overfitting, which can be seen since your training and test error is somewhat similar.
- I like that your feature engineering did not increase the complexity of your dataset too much, since one-hot encoding was only done with features with few categories.

# Three Areas of Improvement
- I am unclear about how you chose which subset of features to use. It may be obvious which features would be the most predictive, but I think backing that up with summary statistics or data exploration would make the argument stronger.
- The bar graphs could use a little more explanation. The acronyms are confusing and I am not sure which features are being included/excluded based on missing values.
- The models you are plan on using in the future could use more justification. Why do you like Bagging and Neural Networks will perform well?

Overall, nice job!
